docs/getting-started/accessing-resources/getting-access-to-data.md

```diff
@@ -54,62 +72,76 @@ See our documentation [getting access to AWS](index.md#getting-access-to-aws) fo
+    ```sh
+    ./download-data.py
+    ```
 
-You can list all the options for the download data script by running the following from the root directory of the repository:
+- If you prefer to work with `AnnData` files, use the `--format` option as follows:
 
-```sh
-./download-data.py --help
-```
+    ```sh
+    ./download-data.py --format AnnData
+    ```
 
-You can run the download script with all default options to download all processed samples from the most recent release in `SingleCellExperiment` format:
+- To download a specific release, you can use the `--release` flag as follows:
 
-!!! warning
-    **The full data release download is quite large: over 35 GB for `SingleCellExperiment` format, and over 100 GB for `AnnData` format.
-    Please download the full data release with caution!**
 
-```sh
-./download-data.py
-```
 
-If you prefer to work with `AnnData` files, use the `--format` option as follows:
+<!--
 
-```sh
-./download-data.py --format AnnData
-```
+TODO: This section needs to be reworked to avoid unexpected downloads. We should recommend --use-release instead.
 
-To review what samples would be downloaded without performing the download yet, you can use the `--dryrun` option as follows:
+These commands will download either test data or results, respectively, and will
 
 ```sh
-./download-data.py --dryrun
+./download-data.py --release current
+
+./download-results.py --release current
 ```
+If you had already downloaded the most recent data or results, this will not repeat downloading the files you already have.
 
-If you're only working with a subset of the data, you can use the `--projects` or `--samples` to download select samples (note: these options are mutually exclusive):
+-->
 
-```sh
-# use the --projects option
-./download-data.py --projects SCPCPXXXXXX
+- To review what samples would be downloaded without performing the download yet, you can use the `--dryrun` option as follows.
+    - We strongly recommend using the `--dryrun` flag the first time you run the script for a new data download to ensure the downloaded files are as expected.
 
-# use the --samples option
-./download-data.py --samples SCPCSXXXXXX,SCPCSXXXXXY
-```
+    ```sh
+    ./download-data.py --dryrun
+    ```
+
+- If you're only working with a subset of the data, you can use the `--projects` or `--samples` to download select projects or samples, respectively (note that these options are mutually exclusive):
+
+    ```sh
+    # use the --projects option
+    ./download-data.py --projects SCPCPXXXXXX
+
+    # use the --samples option
+    ./download-data.py --samples SCPCSXXXXXX,SCPCSXXXXXY
+    ```
+
+- To update the `data/current` symlink, for example if you have downloaded multiple releases and/or [test data](#accessing-test-data), use the `--update-symlink` flag in one of the following ways.
+  - Note that no data will be downloaded if you use this flag.
+    ```sh
```

Comment:
Blank line for form. I have some fear that including the blank line before the code block that I would prefer could break the nesting, or I would have suggested it everywhere. But if you can, I would do it everywhere. 
```suggestion

    ```sh
```